pages/index.js

```diff
@@ -26,7 +26,6 @@ export default function Home() {
 
           <a href="https://nextjs.org/learn" className={styles.card}>
             <h2>Learn &rarr;</h2>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
```

Comment:
comment it.

Reply:
i have solve this in https://github.com/maxjagaapp/nextjs-sample/pull/4/commits/a76da580ee92ca349a6eabdf5a0490bf218c4b96